cloud_connectors/azure/scripts/digital_twins_setup_smart_trailer.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+set -e
+
+# Set the current directory to where the script lives.
+cd "$(dirname "$0")"
+
+# Function to display usage information
+usage() {
+    echo "Usage: $0 [-r|--resource-group-name] <RESOURCE_GROUP_NAME> [-l|--location] <DEPLOYMENT_LOCATION> [-d|--digital-twins-name] <DIGITAL_TWINS_NAME>"
+    echo "Example:"
+    echo "  $0 -r myRG -l westus2 -d myADT"
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]
+do
+key="$1"
```

Comment:
Please consider using indentation for the while loop's body.

Reply:
Indented here and the script I used as base